src/main/java/com/study_buddy/study_buddy/config/SecurityConfig.java

```diff
@@ -17,19 +25,52 @@ public class SecurityConfig {
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         return http
                 .authorizeHttpRequests(auth -> {
-                    // Add endpoint to here to bypass login
-                    // Use this to enable easy testing with Postman
-                    auth.requestMatchers("/", "/login/**", "/example").permitAll();
+                    auth.requestMatchers("/login/register", "/login/**", "/example",
```

Comment:
I think you don't need both /login/register and /login/**, as /login/** matches for everything starting with /login